### PR TITLE
Post Django32 cleanup

### PR DESCRIPTION
**Pull request by aht007**

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -6,36 +6,36 @@
...
-charset-normalizer==2.0.6
+charset-normalizer==2.0.10
     # via requests
-click==8.0.1
+click==8.0.3
     # via cookiecutter
 cookiecutter==1.7.3
     # via -r requirements/base.in
-django==2.2.24
+django==2.2.26
```
Comment (awais786) [inline requirements/base.txt:31]: why it is using django=2.2

Comment (aht007) [reply requirements/base.txt]: Resolved this

File: setup.py [hunk 2]
```diff
@@ -82,23 +121,19 @@ def get_version(*file_paths):
...
         'Framework :: Django :: 3.2',
+        'Framework :: Django :: 4.0',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)',
         'Natural Language :: English',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.8',
     ],
-    tests_require=load_requirements(f'{os.getcwd()}/requirements/test.txt'),
+    tests_require=load_requirements('requirements/test.txt'),
```
Comment (aht007) [inline setup.py:136]: We don't need to get current working directory as we are already at the root

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,6 +4,13 @@ Change history for XBlock SDK
 
 These are notable changes in XBlock.
 
+0.5.0
+-----
+* Removed Django22, 30 and 31 support
+* Added Django40 support
+* Renamed CI job so that the our modernizers work fine
```
Comment (Jawayria) [inline CHANGELOG.rst:11]: No need for 'the' before 'our modernizers'

**Review by UsamaSadiq — APPROVED** [review]

File: setup.py [hunk 4]
```diff
@@ -18,6 +18,58 @@ def find_package_data(pkg, data_paths):
     return {pkg: data}
 
 
+def load_requirements(*requirements_paths):
```
Comment (mraarif) [inline setup.py:21]: there's a shorter version of this method in other packages, we shouldn't be repeating this everywhere

**Review by mraarif — APPROVED** [review]

**Pull request merged**